### PR TITLE
Space after triple equals

### PR DESCRIPTION
**Pull request by anna-zzz**
Little thingy that always bothers me with Scalatest matchers :)
What do you think?

### CODE REVIEW

File: core/src/main/scala/org/scalafmt/internal/Router.scala [hunk 1]
```diff
@@ -282,6 +282,7 @@ class Router(formatOps: FormatOps) {
           } =>
         val modification: Modification = leftOwner match {
           case _: Mod => Space
+          case t: Term if style.spaceBeforeTripleEquals && t.tokens.map(_.code) == Seq("===") => Space
```
Comment (olafurpg) [inline core/src/main/scala/org/scalafmt/internal/Router.scala]: `t: Term.Name`

File: core/src/main/scala/org/scalafmt/ScalafmtStyle.scala [hunk 2]
```diff
@@ -59,6 +59,7 @@ case class ScalafmtStyle(
     spacesInImportCurlyBrackets: Boolean,
     allowNewlineBeforeColonInMassiveReturnTypes: Boolean,
     binPackParentConstructors: Boolean,
+    spaceBeforeTripleEquals: Boolean,
```
Comment (olafurpg) [inline core/src/main/scala/org/scalafmt/ScalafmtStyle.scala]: I suppose `spaceAfterTripleEquals` is more accurate :)

**Comment by olafurpg** [comment]
The first PR that touches `Router` 🎉 Thanks for doing this.

At first I thought there was a bug in scalafmt since I expected the `must === (2)` to be an infix application and arguments to infix applications always get a space. Turns out it's just a regular method call that's passed as an argument to must. I personally don't like how ScalaTest has made this a convention since it makes infix applications even more confusing. I'm happy to include this as a flag since I suspect many users see this differently than me.

File: core/src/main/scala/org/scalafmt/ScalafmtStyle.scala (hunk 2 above)
Comment (anna-zzz) [inline core/src/main/scala/org/scalafmt/ScalafmtStyle.scala]: Oops :)

**Comment by anna-zzz** [comment]
Fantastic!
So I also need to update readme I suppose? Anything else except your comments?

**Comment by olafurpg** [comment]
Right, I forgot, for now it's sufficient to only document this flag in the `ScalafmtStyle` docstring. The cli flags are automatically included in the readme which appears on the website. The plan is add clearer configuration examples to the website soon.

**Comment by olafurpg** [comment]
LGTM, no need to wait for travis since the previous fix passed CI.

**Comment by olafurpg** [comment]
Thanks a lot for this, it's the first non-trivial contribution to scalafmt :) FYI, I don't expect to publish 0.2.6 until in the later part of June as I have to complete on my thesis writing first.

**Comment by olafurpg** [comment]
@anna-zzz Just wanna draw your attention to #303 if you're itching to use the new flag.

**Pull request merged**